lib/iex/lib/iex/info.ex

```diff
@@ -65,11 +65,14 @@ defimpl IEx.Info, for: Atom do
   end
 
   defp format_time({year, month, day, hour, min, sec}) do
-    "#{year}-#{month}-#{day} #{hour}:#{min}:#{sec}"
+    "#{year}-#{zeropad(month)}-#{zeropad(day)} #{zeropad(hour)}:#{zeropad(min)}:#{zeropad(sec)}"
   end
 
   defp default_or_apply(nil, _), do: "no value found"
   defp default_or_apply(data, fun), do: fun.(data)
+
+  defp zeropad(number) when number < 10, do: "0#{number}"
```

Comment:
Maybe we should use same naming as hex uses: https://github.com/hexpm/hex/blob/master/lib/mix/tasks/hex/info.ex#L185.